owl/EASE-SAY.owl

```diff
@@ -63,6 +111,33 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Instrument -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Instrument"/>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Patient -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Patient"/>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#AgentRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#AgentRole">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#classifies"/>
+                <owl:allValuesFrom rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalAgent"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A role often employed in ExecutableSchemas that generalises across thematic relations, such as agent or experiencer, which themselves abstract away from verb-specific roles.</rdfs:comment>
```

Comment:
I do not get the difference between "thematic relations" and "verb-specific roles".
Could you rephrase or elaborate more (e.g. provide an example that showcases difference)?

Reply:
maybe just remove "which themselves abstract away from verb-specific roles"?